spec/views/events/show.html.erb_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
```

Comment:
I don't think we need this test at all. We're certainly going to change the display and not include all of the those fields, as some are mutually exclusive.